SSRN.js

```diff
@@ -123,21 +124,15 @@ var testCases = [
 						"creatorType": "author"
 					}
 				],
-				"notes": [],
-				"tags": [
-					"human rights",
-					"genocide",
-					"Japan",
-					"international law",
-					"nested analysis",
-					"mixed methods",
-					"event history analysis",
-					"survival models",
-					"hazard models",
-					"international norms",
-					"norm cascades"
-				],
-				"seeAlso": [],
+				"date": "2009",
+				"abstractNote": "What explains the large variation in the time taken by different countries to ratify the 1948 Genocide Convention? The costs of ratiﬁcation would appear to be relatively low, yet many countries have waited for years, and even decades, before ratifying this symbolically important treaty. This study employs a \"nested analysis\" that combines a large-n event history analysis with a detailed study of an important outlying case in order to explain the main sources of this variation. The initial event history history produces a puzzling ﬁnding: countries appear to be less likely to ratify the treaty if relevant peer countries have already done so. We use the case of Japan -- which has not yet ratiﬁed the Genocide Convention, despite the predictions of the event history model -- to explore the proposed causes of ratiﬁcation in more detail. Based on these ﬁndings, we suggest that once the norms embodied in a treaty take on a sufﬁciently \"taken-for-granted\" character, many countries decide that the costs of ratiﬁcation outweigh its marginal beneﬁts. The pattern of ratiﬁcation of the Genocide Convention therefore does not appear to ﬁt the classic model of the \"norm cascade\" that has been used to explain the adoption of other human rights norms. We conclude with suggestions for how the validity of our theory could be tested through a combination of further large-n and small-n analysis.",
+				"institution": "Social Science Research Network",
+				"libraryCatalog": "papers.ssrn.com",
+				"place": "Rochester, NY",
+				"reportNumber": "ID 1450387",
+				"reportType": "SSRN Scholarly Paper",
+				"shortTitle": "Who Doesn't Support the Genocide Convention?",
+				"url": "https://papers.ssrn.com/abstract=1450387",
 				"attachments": [
 					{
 						"title": "Full Text PDF",
```

Comment:
I tested this now, but PDFs are still not working. There are various problems why this does not work (and might not work easily):
* The url in the Highwire metadata is just pointing to the article itself, e.g. `<meta name="citation_pdf_url" content="https://papers.ssrn.com/sol3/Delivery.cfm?abstractid=1450387">`.
* The [url given on the website](https://papers.ssrn.com/sol3/Delivery.cfm/SSRN_ID1457896_code1124430.pdf?abstractid=1450387&mirid=1) will answer with a 302 redirect to a temporary url, e.g. https://poseidon01.ssrn.com/delivery.php?ID=701020068111102101093115090073117106050051026007034010028119075074115030099102086074052006096099098123062123085114073109119015121051066022058093067123027007005102080025003000086073067112079109003084076125005103102023110012010105020103092022022025006090&EXT=pdf.
* To catch the 302 redirection we would need to read this information from the header, which I am not sure how to do that...
* After playing a little around with this one example, I receive now an SSRN Alert about an "unusual donwload pattern":
![ssrn-unusual-download-pattern](https://user-images.githubusercontent.com/5199995/27519361-4421d1ac-59f2-11e7-84c6-3cbbf5aca9b4.jpg)